starter_code/app.js

```diff
@@ -4,25 +4,98 @@ const express = require('express');
 const hbs = require('hbs');
 
 // require spotify-web-api-node package here:
-
+const SpotifyWebApi = require("spotify-web-api-node");
 
 
 const app = express();
 
 app.set('view engine', 'hbs');
 app.set('views', __dirname + '/views');
 app.use(express.static(__dirname + '/public'));
-
+hbs.registerPartials(__dirname + '/views/partials');
 
 // setting the spotify-api goes here:
-
-
-
-
-
-
+const spotifyApi = new SpotifyWebApi({
+    clientId: process.env.CLIENT_ID,
+    clientSecret: process.env.CLIENT_SECRET
+  });
+  
+  
 // the routes go here:
 
 
+app.get('/' , function(req , res , next) {
+    res.render('index');
+});
+
+app.get('/artists' , function(req , res , next) {
```

Comment:
Pas besoin de rappeler le token, vous pouvez faire plus simple : 
app.get('/artists', function (req, res, next) {
  spotifyApi
  .searchArtists(`${req.query.artist}`)
  .then(data => {
    console.log("The received data from the API: ", data.body);
    res.render('artists', {
      artists: data.body.artists.items
    });
  })
  .catch(err => {
    console.log("The error while searching artists occurred: ", err);
  });
});